ConsoleApplicationGoBoard/ConsoleFunctions.cpp

```diff
@@ -0,0 +1,87 @@
+#include "ConsoleFunctions.h"
+#include <iostream>
+#include <string>
+
+using namespace std;
```

Comment:
you should prefix the "std" with two colons ("using namespace ::std")... this is to avoid problems with nested namespaces and qualifies it as the root namespace